### DISCUSSION TITLE
We can add a new property: `user-select`

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#1377] [Ideas & RFCs]
Without `user-select`, if the text when clicked upon gets selected (let's say inside a button-like text, _which is not expected_).
![image](https://github.com/fastn-stack/fastn/assets/118595104/2225d74c-c60e-4030-b121-205e1fc3e560)

But if we do not want the text to be selected, we should have a property `user-select` with the value `none`. 
![image](https://github.com/fastn-stack/fastn/assets/118595104/75abf07f-69bb-4627-a39e-bc06cc4297a3)

**Comment by amitu** [comment]
Its not supported in Safari: https://developer.mozilla.org/en-US/docs/Web/CSS/user-select :-(

We only add something if it is supported by 98% or so.
